src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/returncount/Example1.java

```diff
@@ -0,0 +1,44 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="ReturnCount">
+      <property name="max" value="3"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.returncount;
+
+// xdoc section -- start
+import java.util.function.Predicate;
```

Comment:
Import is not required in html website rendering.
Please move comment above import to be right above class

Reply:
Done